woocommerce/woocommerce-android
# Issue/3726 Fixed horizontal margins for payment collection flow

Parent issue #3726

I noticed that horizontal margins are not correct for the collection flow. The PR fixes this

### How to test

* Open Order in USD
* Click collect money
* CHeck the horizontal margins for the buttons are bigger now (as per design)

https://user-images.githubusercontent.com/4923871/124079140-741a0300-da51-11eb-9e7c-bec3c187a5bf.mp4
